### ISSUE TITLE
client.d.ts is being generated with two copies of every function

### ISSUE THREAD

**Issue by dfabulich** [ceifa/steamworks.js#83]
In my PR branch #82, I've found that client.d.ts is being generated in a very bad way.

`client.d.ts` should start like this:

```ts
export function init(appId: number): void
export function restartAppIfNecessary(appId: number): boolean
export function runCallbacks(): void
export interface PlayerSteamId {
  steamId64: bigint
  steamId32: string
  accountId: number
}
export namespace achievement {
  export function activate(achievement: string): boolean
  export function isActivated(achievement: string): boolean
}
```

But when I `git clean -xdf && npm install && npm run build:debug` in my branch, I find that `client.d.ts` is changing, which it shouldn't, and it's changing in a bad way:

```ts
export function init(appId: number): void
export function restartAppIfNecessary(appId: number): boolean
export function runCallbacks(): void
export interface PlayerSteamId {
  steamId64: bigint
  steamId32: string
  accountId: number
}
export function init(appId: number): void
export function restartAppIfNecessary(appId: number): boolean
export function runCallbacks(): void
export interface PlayerSteamId {
  steamId64: bigint
  steamId32: string
  accountId: number
}
export namespace achievement {
  export function activate(achievement: string): boolean
  export function isActivated(achievement: string): boolean
  export function activate(achievement: string): boolean
  export function isActivated(achievement: string): boolean
}
```

**Comment by dfabulich** [comment]
So, I ran another `git bisect`; this time, I manually reapplied my patch from PR #82 at each bisect point. And I found that the revision causing this is 4c9ce0166520ddcfaa2d568d4cb8601309a2202b.

That revision fails to build due to #80, but if I manually reapply my patch from #82 to that revision, it builds, and generates a corrupted `client.d.ts` file. When I check out its parent revision, e7260183760e91d025fe09495f87051c3c522b29, and fix its build with the patch from #82, it generates a normal, healthy-looking `client.d.ts` file.

The problem is due to bumping to the latest version of `@napi-rs/cli`. 4c9ce0166520ddcfaa2d568d4cb8601309a2202b upgraded from `@napi-rs/cli` 2.14.8 to 2.15.2. Downgrading to 2.14.8 seems to fix it.
